README.md

```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
 ```
 
-> Note: If you are using macOS please clone the [Polkadot repo](https://github.com/paritytech/polkadot) and run it locally. At the moment there is no `polkadot` binary for MacOs.
+> Note: If you are using macOS please clone the [polkadot-sdk repo](https://github.com/paritytech/polkadot-sdk) and run it locally. At the moment there is no `polkadot` binary for MacOs.
 
 The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. 
 At the end of the download, the `setup` script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example:
 
 ```bash
-Please add the dir to your $PATH by running the command: export PATH=/home/<user>/zombienet/dist:$PATH
+Please add the dir to your $PATH by running the command: export PATH=/<full path to the project>/zombienet/dist:$PATH
 ```
 
 ### Build adder-collator (needed for running examples with native provider)
 
 You can build it from source like this
 
 ```bash
-git clone git@github.com:paritytech/polkadot
-cd polkadot
-cargo build --profile testnet -p test-parachain-adder-collator
-export PATH=$(pwd)/target/testnet:$PATH
+❯ git clone git@github.com:paritytech/polkadot-sdk.git
```

Comment:
polkadot repo archived and project moved to polkadot-sdk repo, [see](https://github.com/paritytech/polkadot/blob/52209dcfe546ff39cc031b92d64e787e7e8264d4/README.md?plain=1#L3)

applicable to all similar changes around this PR